com.unity.mobile.notifications/Runtime/Android/AndroidNotificationExtension.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using UnityEngine;
+
+namespace Unity.Notifications.Android
+{
+    internal static class AndroidNotificationExtension
```

Comment:
Usually it's AndroidNotificationExtension**s**

Reply:
Fixed at https://github.com/Unity-Technologies/com.unity.mobile.notifications/pull/33/commits/452583442cc6210d66935fc66d6a68c606a91a20.